solr/core/src/test/org/apache/solr/cloud/FullSolrCloudDistribCmdsTest.java

```diff
@@ -272,6 +272,40 @@ public void testDeleteByIdImplicitRouter() throws Exception {
     }
   }
 
+  public void testRTGCompositeRouterWithRouterField() throws Exception {
+    final CloudSolrClient cloudClient = cluster.getSolrClient();
+    final String testCollectionName =
+        "composite_collection_with_routerfield_" + NAME_COUNTER.getAndIncrement();
+    assertEquals(
+        RequestStatusState.COMPLETED,
+        CollectionAdminRequest.createCollection(testCollectionName, "_default", 2, 2)
+            .setRouterName("compositeId")
+            .setRouterField("routefield_s")
+            .setShards("shard1,shard2")
+            .processAndWait(cloudClient, DEFAULT_TIMEOUT));
+    ZkStateReader.from(cloudClient)
+        .waitForState(
+            testCollectionName,
+            DEFAULT_TIMEOUT,
+            TimeUnit.SECONDS,
+            (n, c1) -> DocCollection.isFullyActive(n, c1, 2, 2));
+
+    // Add six documents w/diff routes (all sent to shard1 leader's core)
```

Comment:
four?

Reply:
thanks Eric. copy pasta strikes again :) will fix it